client/src/App.js

```diff
@@ -1,14 +1,19 @@
 import React, { Fragment } from "react";
+import { Router } from "@reach/router";
 import { createGlobalStyle } from "styled-components";
-
+import Home from "Components/Home";
+import LocationHome from "Components/LocationHome";
```

Comment:
As there is more than one component, you can create an index file that exports them, so there would be one import like: import { Home, LocationHome } from 'Components';

Reply:
Great tip, thanks! :)